velox/functions/prestosql/HyperLogLogFunctions.h

```diff
@@ -48,7 +48,7 @@ struct EmptyApproxSetFunction {
 
   FOLLY_ALWAYS_INLINE bool call(out_type<HyperLogLog>& result) {
     static const std::string kEmpty =
-        common::hll::SparseHll::serializeEmpty(11);
+        common::hll::SparseHll::serializeEmpty(12);
```

Comment:
I assume that 11 bits are sufficient 0.023 error, but 12 are need for 0.01625. Please, let me know if this change is about something else.

Reply:
Exactly.  I also have tests to check 0.023 converting to 11 bits and 0.01625 to 12 bits